src/widgets/HistogramWidget.js

```diff
@@ -8,21 +8,21 @@ import { getHistogram } from './models';
 import { AggregationTypes } from './AggregationTypes';
 
 /**
-  * Renders a <HistogramWidget /> component
-  * @param  props
-  * @param  {string} props.id - ID for the widget instance.
-  * @param  {string} props.title - Title to show in the widget header.
-  * @param  {string} props.dataSource - ID of the data source to get the data from.
-  * @param  {string} props.column - Name of the data source's column to get the data from.
-  * @param  {string} props.operation - Operation to apply to the operationColumn. Must be one of those defined in `AggregationTypes` object.
-  * @param  {number[]} props.ticks - Array of thresholds for the X axis.
-  * @param  {formatterCallback} [props.xAxisformatter] - Function to format X axis values.
-  * @param  {formatterCallback} [props.formatter] - Function to format Y axis values.
-  * @param  {boolean} [props.viewportFilter=false] - Defines whether filter by the viewport or globally. 
-  * @param  {boolean} [props.tooltip=true] - Whether to show a tooltip or not
-  * @param  {errorCallback} [props.onError] - Function to handle error messages from the widget.
-  * @param  {Object} [props.wrapperProps] - Extra props to pass to [WrapperWidgetUI](https://storybook-react.carto.com/?path=/docs/widgets-wrapperwidgetui--default)
-  */
+ * Renders a <HistogramWidget /> component
+ * @param  props
+ * @param  {string} props.id - ID for the widget instance.
+ * @param  {string} props.title - Title to show in the widget header.
+ * @param  {string} props.dataSource - ID of the data source to get the data from.
+ * @param  {string} props.column - Name of the data source's column to get the data from.
+ * @param  {string} props.operation - Operation to apply to the operationColumn. Must be one of those defined in `AggregationTypes` object.
+ * @param  {number[]} props.ticks - Array of thresholds for the X axis.
+ * @param  {formatterCallback} [props.xAxisformatter] - Function to format X axis values.
+ * @param  {formatterCallback} [props.formatter] - Function to format Y axis values.
+ * @param  {boolean} [props.viewportFilter=false] - Defines whether filter by the viewport or not.
```

Comment:
I'm not sure this improves the definition for this parameter. I suggest to keep "globally" or explain a bit the other option.

Reply:
'globally' LGTM